code/__DEFINES/mobs.dm

```diff
@@ -135,7 +137,7 @@
 #define BODYPART_ID_LARVA "larva"
 #define BODYPART_ID_PSYKER "psyker"
 #define BODYPART_ID_MEAT "meat"
-
+#define BODYPART_ID_PEG "peg"
```

Comment:
```suggestion
#define BODYPART_ID_PEG "peg"

```
put that newline back

Reply:
done, can you resolve this the message yelling at me is bothering me and my perfect checks finished